### ISSUE TITLE
Add Dynogels specific errors for create, get and destroy

### ISSUE THREAD

**Issue by svanzoest** [clarkie/dynogels#100]
Produce dynogels-specific errors #73 was closed, however, #74 only handled `update`.

**Comment by clarkie** [comment]
Happy to take a PR if you have time?

Thanks

Clarkie

**Comment by dangerginger** [comment]
Is this just a matter of rewriting the `return` statement on the create, get and destroy methods?  Are there any side effects to doing this?  @cdhowie?

**Comment by cdhowie** [comment]
The only side effect would be code that depends on the existing structure of errors.  For example, in #74 I changed the message from a JSON string to a human-readable string, and instead put the previously-stringified object onto the error object.

Existing code using dynogels might try to parse the error message as JSON, and that would fail after changing.  However, I would argue that parsing error messages is a fragile approach to begin with.

tl;dr: It's technically backwards-incompatible, but I don't think we should care in this instance.
